p2p/starknet/starknet_test.go

```diff
@@ -53,15 +55,104 @@ func TestClientHandler(t *testing.T) {
 	}, testNetwork, log)
 
 	t.Run("get block headers", func(t *testing.T) {
-		res, cErr := client.RequestBlockHeaders(testCtx, &spec.BlockHeadersRequest{})
+		type pair struct {
+			header      *core.Header
+			commitments *core.BlockCommitments
+		}
+		pairsPerBlock := []pair{}
+		for i := 0; i < 2; i++ {
+			ui := uint64(i)
```

Comment:
Consider `for i := uint64(0)...`, so there is no need to create another variable.